### ISSUE TITLE
Configure System Folders IMAP Special use extension no folder to select

### ISSUE THREAD

**Issue by chrisbcarlson** [bkaankose/Wino-Mail#48]
Looking for any guidance on this issue. Thank you

![Screenshot 2023-11-13 143004](https://github.com/bkaankose/Wino-Mail/assets/47155595/e30fdb7e-02b2-4d47-a401-4167b6555d82)

**Comment by chrisbcarlson** [comment]
I apologize I see there is already a similar issue for this. https://github.com/bkaankose/Wino-Mail/issues/44

**Comment by bkaankose** [comment]
Thanks for the report.

That one is actually more about the height of the dialog, but I can confirm the issue you are experienced as well. I will address this in the next hotfix release I'm preparing for upcoming weeks.

**Comment by Benjin** [comment]
I'm hitting this as well; looking forward to the update!

**Comment by bkaankose** [comment]
Version 1.6.7 is released. Please try the new version out and let me know if you are still experiencing this issue :)

**Comment by chrisbcarlson** [comment]
Hi, I have not done to much troubleshooting. I just wanted to give you feedback right away. But I am still running into an issue. It seems different now. Here is a screenshot. This is what it gives me when I click on "click here to fix it"
![image](https://github.com/bkaankose/Wino-Mail/assets/47155595/60235078-f986-4e1d-875b-8a8942158762)

**Comment by bkaankose** [comment]
You should not see this popup anymore once you have configured everything.

Is this server custom one or any public server that I can create account for free?

**Comment by chrisbcarlson** [comment]
Roundcube on a plesk server. Thunderbird works. I am sure this is not a common use case. So prioritize as needed. Thank you

**Comment by bkaankose** [comment]
Thank you.

It's weird that your Inbox is not detected. That's why you see this pop up all the time regardless of other folder configurations you have. Inbox should always be available once the authentication is done. This might also be a bug in [Mailkit](https://github.com/jstedfast/MailKit) library.

I'll do some investigations around it :)

**Comment by Benjin** [comment]
I am no longer able to set up my connection at all.  I am using ProtonMail via their Proton Mail Bridge app (runs a local IMAP/SMTP server that handles the encryption).  If you want, you can set this up for free to test.

This _does_ work via the built-in Mail app's advanced dialog, so I've confirmed that the server is working with the input I'm plugging into WinoMail.

**Comment by Vinsup** [comment]
Same issue here on my side with my email @orange.fr.

**Comment by Vinsup** [comment]
Hey,
Go the update today.
I was able to "save the configuration" and the error messages went away, but still no "INBOX" folder...
![image](https://github.com/bkaankose/Wino-Mail/assets/38085235/fbf8143c-f44c-436e-9900-8ea0b5d1e5f8)

**Comment by EvilVir** [comment]
I have very same issue. Using Nazwa.pl provider.

When I was configuring Thunderbird with this account I saw something strange, so maybe that would lead to some solution. So after initial configuration in Thunderbird folders like "Trash", "Draft" etc. were displayed as subfolders of "Inbox". I had to put "INBOX." in the "Personal namespace" configuration field and "INBOX" in the "IMAP server directory" to make it work properly. Screenshots below.

Problem in Thunderbird:
<img width="154" alt="Screenshot 2024-01-13 185907" src="https://github.com/bkaankose/Wino-Mail/assets/2843963/7ba60109-0d97-4d58-9e50-220d19580043">

Thunderbird settings to solve it:
<img width="368" alt="Screenshot 2024-01-13 185952" src="https://github.com/bkaankose/Wino-Mail/assets/2843963/c0d20ebc-f985-4630-bf31-0edf7de6fd47">

WinoMail view of same account:
<img width="233" alt="Screenshot 2024-01-13 190157" src="https://github.com/bkaankose/Wino-Mail/assets/2843963/cc5d063e-6c2c-453f-971b-6baeb0ed2b75">


Hope this helps solving the issue. I would be happy to look into this if you'd give me access to the code. Also I can translate the app to Polish if you want.

**Comment by Eikwerno** [comment]
I am having the same issue for a domainfactory (https://www.df.eu/) mail server.

**Comment by Rahsna** [comment]
Yeap. Same isssue here.
Also using roundcube - open source webmail software.
I am guessing it is an issue with the open source webmail software and Wino is activating some bug or the software has a weird way of handling special e-mail folders?

Going into roundcube > Preferences > Special Folders: Main Options, _Show real names for special folders_ is off by default. 

My account in Wino mail is also missing the INBOX folder.
<img width="234" alt="image" src="https://github.com/bkaankose/Wino-Mail/assets/35332469/b9828103-e98c-4348-aa0a-914e8e4b4bba">

Interestingly the Inbox folder is not listed under special folders in roundcube settings.
Furthermore, some of the subfolders are displayed twice:
<img width="236" alt="image" src="https://github.com/bkaankose/Wino-Mail/assets/35332469/bdf6b9ec-bda4-4ca4-8782-5c2fa574aad5">
First as actual subfolders, then as non-subfolders.
Sorry if that is another issue, I am simply trying to give enough data to properly debug this :)

**Comment by bkaankose** [comment]
This issue with invisible Inbox is fixed for the next release, which have the source code that I will open source the app with.

Main issue was Inbox being the root folder and the first/only namespace available. In that case, Wino had a bug to ignore the Inbox since namespaces were marked to be ignored.

It's fixed by somebody who provided me a test account in his/her own server :) For the new version I will open a mailing list in the Discord channel for people to participate the beta version. Changes are too big for myself to publish it without doing the user testing first.

If you'd like to test the new beta version there please track the updates on the Discord link. I will share some details there soon.
Thanks everybody for the help to diagnose this!

**Comment by EvilVir** [comment]
@bkaankose sorry to say but it is not fixed for me. Furthermore account is not added at all after long wait time on the "Preparing folders" screen.

If you want I can set you up test account to see this issue.

**Comment by bkaankose** [comment]
> @bkaankose sorry to say but it is not fixed for me. Furthermore account is not added at all after long wait time on the "Preparing folders" screen.
> 
> If you want I can set you up test account to see this issue.

Did you give beta version a try? If yes, please provide me a test account so I can check myself.

**Comment by EvilVir** [comment]
E-mail sent to you with account details. And yes I tried 1.7.1 installed alongside 1.6.12.

**Comment by bkaankose** [comment]
@EvilVir Could you please retry with the latest version 1.7.6?

**Comment by EvilVir** [comment]
I will check on Monday - currently I’m on offline vacations :)

**Comment by bkaankose** [comment]
This issue should be fixed in the latest version 1.9.0. Please re-open if you still experience it.

**Comment by mtsbh** [comment]
Hello! I'm using 1.9.0 and can't add my email.

![image](https://github.com/user-attachments/assets/c9f645a3-a366-457d-a4e7-66399eacf1f5)
